src/firefly/js/templates/lightcurve/LcUtil.jsx

```diff
@@ -16,6 +16,23 @@ export function getTypeData(key, val='', tip = '', labelV='', labelW) {
     };
 }
 
+/**
+ * Returns only numerical column names form raw lc table
+ * @param {Object} rawTbl
+ * @returns {TableColumn[]} - array of table columns objects
```

Comment:
This is wrong.  It's returning string[] - array of column names, not TableColumn[].